aibolit/patterns/method_chaining/method_chaining.py

```diff
@@ -0,0 +1,75 @@
+import javalang
+
+import uuid
+from collections import defaultdict
+
+
+class MethodChainFind:
+
+    def __init__(self):
+        pass
+
+    def traverse_node(self, node, dict_with_chains, uuid_method):
```

Comment:
@lyriccoder This method should be private, since it shouldn't be accessed from outside this class